src/PrestaShopBundle/Resources/views/Admin/Module/manage.html.twig

```diff
@@ -57,32 +57,32 @@
                            <span class="help-box" data-toggle="popover"
-                                data-title="{{ "Installed modules"|trans({}, 'AdminModules') }}"
-                                data-content="{{ "These are all the modules you’ve added to your shop, either by buying it from PrestaShop Addons, or by uploading it directly."|trans({}, 'AdminModules') }}">
+                                data-title="{{ "Installed modules"|trans({}, 'Admin.Modules') }}"
+                                data-content="{{ "These are all the modules you’ve added to your shop, either by buying it from PrestaShop Addons, or by uploading it directly."|trans({}, 'Admin.Modules') }}">
                            </span>
                    </div>
 		{% include 'PrestaShopBundle:Admin/Module/Includes:grid.html.twig' with { 'modules' : modules.modules, 'display_type' : 'list', 'origin' : 'manage'  } %}
 
 		<hr class="top-menu-separator">
 
 		<div class="module-short-list">
-			<span class="module-search-result-wording">{{modules.native_modules|length}} built-in modules</span>
+			<span class="module-search-result-wording">{{ '%nbModules% built-in modules'|trans({'%nbModules%' : modules.native_modules|length}, 'Admin.Modules') }}</span>
                            <span class="help-box" data-toggle="popover"
-                                 data-title="{{ "Built-in modules"|trans({}, 'AdminModules') }}"
-                                data-content="{{ "These modules from PrestaShop are preinstalled when you set-up your shop. They cover the basics of e-commerce and comes for free."|trans({}, 'AdminModules') }}">
+                                 data-title="{{ "Built-in modules"|trans({}, 'Admin.Modules') }}"
+                                data-content="{{ "These modules from PrestaShop are preinstalled when you set-up your shop. They cover the basics of e-commerce and comes for free."|trans({}, 'Admin.Modules') }}">
                            </span>
 		</div>
 		{% include 'PrestaShopBundle:Admin/Module/Includes:grid.html.twig' with { 'modules' : modules.native_modules, 'display_type' : 'list', 'origin' : 'manage'  } %}
 
 		<hr class="top-menu-separator">
 
 		<div class="module-short-list">
-			<span class="module-search-result-wording">{{modules.theme_bundle|length}} theme modules</span>
+			<span class="module-search-result-wording">{{ '%nbModules% theme modules'|trans({'%nbModules%' : modules.theme_bundle|length}, 'Admin.Modules') }}</span>
                            <span class="help-box" data-toggle="popover"
-                                data-title="{{ "Theme modules"|trans({}, 'AdminModules') }}"
-                                data-content="{{ "Each theme you install will come with its own set of modules. You’ll find here all the modules related to your active theme."|trans({}, 'AdminModules') }}">
+                                data-title="{{ "Theme modules"|trans({}, 'Admin.Modules') }}"
```

Comment:
one time double quotes, one time single quotes, there will be any standarization or it's up to us which one to use?